AlphaWallet/Tokens/ViewModels/TokensViewModel.swift

```diff
@@ -111,6 +113,9 @@ class TokensViewModel {
                         return $0.type == .erc721
                     } else if keyword.lowercased() == "erc875" || keyword.lowercased() == "erc 875" {
                         return $0.type == .erc875
+                    } else if keyword.lowercased() == "tokenscript" {
```

Comment:
@hboon better to ignore casing? https://stackoverflow.com/questions/30532728/how-to-compare-two-strings-ignoring-case-in-swift-language

Reply:
Yes. It is already using keyword.lowercased()